ledger/metrics/src/main/scala/com/daml/telemetry/TelemetryContext.scala

```diff
@@ -82,35 +79,19 @@ trait TelemetryContext {
 
 }
 
-object TelemetryContext {
```

Comment:
There are a few more methods that are not being used in this repo (yet?), which require more effort to get rid of. Presumably by introducing more inheritance in the oem integration kit repo, which I'm not happy about. I won't address it in this PR.